projects/core/src/i18n/i18next/i18next-translation.service.ts

```diff
@@ -73,16 +91,21 @@ export class I18nextTranslationService implements TranslationService {
 
   /**
    * Returns a fallback value in case when the given key is missing
-   * @param key
+   * @param keys
    */
-  protected getFallbackValue(key: string): string {
-    return isDevMode() ? `[${key}]` : this.NON_BREAKING_SPACE;
+  protected getFallbackValue(keys: string[]): string {
```

Comment:
Should we mention this method's signature change in the 6.0 migration docs as well?